setup.py

```diff
@@ -89,7 +87,7 @@ def __call__(self):
                       "xgboost>=1.5.2,<2.0", "lightgbm>=3.3.2", "shap>=0.45", "pyarrow>=7.0.0",
                       "ipython>=7.32.0", "optuna>=2.10.0", "numpy>=1.26"],
 
-    python_requires=">=3.8",
+    python_requires=">=3.9",
```

Comment:
shouldn't this be 
```suggestion
    python_requires=">3.9",
```
?